lib/controllers/app/account_controller.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:get/get.dart';
+import 'package:pispapp/controllers/ephemeral/account_dashboard_controller.dart';
+import 'package:pispapp/models/account.dart';
+import 'package:pispapp/repositories/interfaces/i_account_repository.dart';
+
+class AccountController extends GetxController {
+  AccountController(this._accountRepository);
+  var accounts = <Account>[].obs;
+
+  IAccountRepository _accountRepository;
+
+  void getAllLinkedAccounts() {
+    accounts.value = _accountRepository.getUserAccounts();
+    Get.find<AccountDashboardController>()
+        .setSelectedAccount(accounts.value[0]);
```

Comment:
Is it necessary to put this here? I think the function does more than what it says in the function name...
You may want to considering changing the function name if you need to perform this here.

Reply:
i am sorry. I actually fixed this in the PR following this. Should have mentioned it.

Reply:
okay, got it, let's keep this unresolved for tracking in the next PR

Reply:
addressed in https://github.com/mojaloop/pisp-demo-app-flutter/pull/12